src/encoded/tests/data/inserts/experiment.json

```diff
@@ -10,7 +10,7 @@
         "status": "released",
         "date_submitted": "2015-01-20",
         "date_released": "2016-01-01",
-        "internal_tags":["DREAM", "ENCORE"],
+        "internal_tags": ["DREAM"],
```

Comment:
Why remove "ENCORE"?

Reply:
Because I made other test experiments, and this one didn’t really fit ENCORE experiments — someone (me maybe) just added ENCORE to this experiment back in the day.